lib/kimono_labs_client/web_client.rb

```diff
@@ -0,0 +1,31 @@
+module KimonoLabsClient
+  # Web client for fetching a json endpoint's content via http.
+  class WebClient
+    attr_reader :api_key
+
+    # initializer
+    def initialize(api_key)
+      fail 'API key must be provided!' if api_key.nil?
```

Comment:
This isn't very confident ruby. If we have argument/options validation a that can raise an exception let's consider a private method that asserts a valid configuration. It should be a predicate method, and it should be private.